scripts/Deploy-MSBuild.ps1

```diff
@@ -8,15 +8,30 @@ Param(
   [string] $runtime = "Desktop"
 )
 
+Set-StrictMode -Version "Latest"
+$ErrorActionPreference = "Stop"
```

Comment:
Wishlist item that may be too hard to implement: Building out the error functionality to delete the folder that was generated for this particular build, or essentially "reset" to the original binaries.

Reply:
I restore original binaries by adding the VS msbuild bin dir under git. If I were to implement restoration of original binaries, that's how I would do it. But I'd rather not do it in this PR :)